tutorials/tts/Fastpitch_Training_GermanTTS.ipynb

```diff
@@ -0,0 +1,1112 @@
+    "\n",
+    "hifigan_manifest_path = dataset_base_path+folder_name+\"/hifigan_\"+dataset_part+\"_ft.json\"\n",
+    "\n",
+    "with open(hifigan_manifest_path, \"w\") as f:\n",
+    "    for r in records:\n",
+    "        f.write(json.dumps(r) + '\\n')\n",
+    "```\n",
+    "\n",
+    "Repeat the above script for train and validation datasets as well. Finally the `/Data/synmels` will look like:\n",
+    "```\n",
+    "/Data/synmels/:\n",
+    "hifigan_test_ft.json\n",
+    "hifigan_train_ft.json\n",
+    "hifigan_val_ft.json\n",
+    "test\n",
+    "train\n",
+    "val\n",
+    "\n",
+    "/Data/synmels/test:\n",
+    "mel_0.npy\n",
+    "mel_1.npy\n",
+    "...\n",
+    "\n",
+    "/Data/synmels/train:\n",
+    "mel_0.npy\n",
+    "mel_1.npy\n",
+    "...\n",
+    "\n",
+    "/Data/synmels/val:\n",
+    "mel_0.npy\n",
+    "mel_1.npy\n",
+    "...\n",
+    "```\n",
+    "\n",
+    "Example hifigan manifest:\n",
+    "```yaml\n",
+    "{\"audio_filepath\": \"/Data/thorsten-de/wavs/e50eb02c25353f85549900d2fc1e0e32.wav\", \"duration\": 2.409977, \"text\": \"Geht die Schandtat auf sein Konto?\", \"normalized_text\": \"Geht die Schandtat auf sein Konto?\", \"mel_filepath\": \"/Data/synmels/test/mel_0.npy\"}\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e00d26ff",
+   "metadata": {},
+   "source": [
+    "## Launch finetuning:\n",
+    "\n",
+    "We will be re-using the existing hifigan config. and hifigan pretrained on english. \n",
+    "\n",
+    "```bash\n",
+    "cd /Data\n",
+    "wget --content-disposition https://api.ngc.nvidia.com/v2/models/nvidia/nemo/tts_hifigan/versions/1.0.0rc1/zip -O tts_hifigan_1.0.0rc1.zip\n",
+    "unzip tts_hifigan_1.0.0rc1.zip\n",
+    "\n",
+    "cd /NeMo\n",
+    "python examples/tts/hifigan_finetune.py --config-path conf/hifigan --config-name hifigan.yaml \\\n",
+    "    model.max_steps=100000 \\\n",
+    "    model.optim.lr=0.00001 \\\n",
+    "    model.train_ds=train_ds_finetune \\\n",
+    "    model.validation_ds=val_ds_finetune \\\n",
+    "    ~model.optim.sched \\\n",
+    "    train_dataset=/Data/synmels/hifigan_train_phonemes_ft.json \\\n",
+    "    validation_datasets=/Data/synmels/hifigan_val_phonemes_ft.json \\\n",
+    "    exp_manager.exp_dir=/result \\\n",
+    "    +init_from_nemo_model=/Data/tts_hifigan.nemo \\\n",
+    "    trainer.devices=-1 \\\n",
+    "    exp_manager.create_wandb_logger=true \\\n",
+    "    exp_manager.wandb_logger_kwargs.name=<wandb_run_name> \\\n",
+    "    exp_manager.wandb_logger_kwargs.project=<wandb_project_name>\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e8d5407b",
+   "metadata": {},
+   "source": [
+    "## Evaluating Fastpitch and Finetuned HiFi-GAN"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fe21406f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hfg_path = \"<path_to_finetuned_hifigan_nemo_or_ckpt>\"\n",
```

Comment:
Can you either make the HiFi-GAN training cell executable or these two not executable? The following cells will only ever be runnable if the previous command has been run, anyway.

Reply:
makes sense. I am making them runnable. Assuming the user will go through and finetune on their own.